library/core/src/ops/bit.rs

```diff
@@ -68,6 +68,17 @@ macro_rules! not_impl {
 
 not_impl! { bool usize u8 u16 u32 u64 u128 isize i8 i16 i32 i64 i128 }
 
+#[stable(feature = "not_never", since = "1.58.0")]
```

Comment:
It looks like this PR missed 1.58.
```suggestion
#[stable(feature = "not_never", since = "1.59.0")]
```